cpp/tests/copying/split_tests.cpp

```diff
@@ -31,6 +31,7 @@
 #include <cudf/utilities/type_dispatcher.hpp>
 
 #include <rmm/device_buffer.hpp>
+#include <rmm/mr/device/pool_memory_resource.hpp>
```

Comment:
I don't think this is being used here?

Reply:
removed

Reply:
Could you see if it is possible to remove the `type_dispatcher.hpp` include as well?  There is no type-dispatch logic in this file and that header file is fairly complex.

Reply:
Missed this comment, I am rebuilding with the change.

Reply:
Fixed here: https://github.com/rapidsai/cudf/pull/13260/commits/7a73d57ad4330a052a21ae9bb3d5cccba4f569bb